virtcontainers/sandbox_test.go

```diff
@@ -1527,3 +1528,35 @@ func TestGetSandboxCpuSet(t *testing.T) {
 		})
 	}
 }
+
+func TestSandboxStoreClean(t *testing.T) {
+	contID := "SandboxStore"
+	contConfig := newTestContainerConfigNoop(contID)
+	hConfig := newHypervisorConfig(nil, nil)
+	assert := assert.New(t)
+
+	// create a sandbox
+	p, err := testCreateSandbox(t, testSandboxID, MockHypervisor, hConfig, NoopAgentType, NetworkConfig{}, []ContainerConfig{contConfig}, nil)
+	assert.NoError(err)
+	defer cleanUp()
+
+	l := len(p.GetAllContainers())
+	assert.Equal(l, 1)
+
+	// persist to disk
+	err = p.storeSandbox()
+	assert.NoError(err)
+
+	loadSandboxConfigFromOldStore(context.Background(), p.ID())
+
+	runtimSidPath := store.SandboxConfigurationRoot(p.ID())
+	runtimSidPath = strings.TrimPrefix(runtimSidPath, "file://")
+
+	err = p.Delete()
+	assert.NoError(err)
+
+	// expect runtimSidPath not exist, if exist, it means this case failed.
+	if _, err := os.Stat(runtimSidPath); err == nil || os.IsExist(err) {
```

Comment:
I think it would be clearer to write this as separate tests:

```go
err := os.Stat(runtimSidPath)
assert.Error(err)
assert.True(os.IsNotExist(err))
```